### PR TITLE
Add oak9cli 0.11.16

### PR DESCRIPTION
**Pull request by awright-oak9**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ x] `brew audit --cask --online <cask>` is error-free.
- [x ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x ] `brew audit --new-cask <cask>` worked successfully.
- [ x] `brew install --cask <cask>` worked successfully.
- [ x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you @awright-oak9, but from our [acceptable Casks documentation](https://docs.brew.sh/Acceptable-Casks#rejected-casks):

> The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/Homebrew/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) as a formula that builds from source. If it is rejected, you may then try again as a cask (link us to the issue so we can see the discussion and reasoning for rejection).

**Comment by awright-oak9** [comment]
@miccal This is not an open-source project. Do I still need to attempt to submit it to core?

**Comment by miccal** [comment]
Thank you for your submission @awright-oak9, but we cannot accept it at this time because it does not meet our [minimum notability requirements](https://docs.brew.sh/Acceptable-Casks#rejected-casks):

> + The Cask is for an app that is too obscure. Examples:
>   + An app from a code repository that is not notable enough (under 30 forks, 30 watchers, 75 stars).

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if it gets notable enough we would then be happy to consider adding this to `homebrew-cask`.

**Comment by bevanjkay** [comment]
We can skip the notability check here, because the repository only hosts the binaries.

**Comment by bevanjkay** [comment]
```
Error: Failure while executing; `/usr/bin/env ditto -x -k /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip /private/tmp/d20230223-11648-ormirk` exited with 1. Here's the output:
ditto: Couldn't read PKZip signature
```

**Comment by awright-oak9** [comment]
@bevanjkay Do you know what causes this error?

**Comment by miccal** [comment]
@awright-oak9 we need a stable `url`:
```
==> Downloading https://github.com/oak9io/oak9.cli-public/blob/master/binaries/iac-mac_os-0.11.16.zip
Downloaded to: /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip
SHA256: 521ea9782e66e79d73e1765494cddc5b08db90483d36c98191ed9a572d67dcb1
==> Retrying download in 2s... (4 tries left)
==> Downloading https://github.com/oak9io/oak9.cli-public/blob/master/binaries/iac-mac_os-0.11.16.zip
Downloaded to: /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip
SHA256: 57865bf528764b6b24975ea180770339020038a4f794dd224fda86fec97fca71
==> Retrying download in 4s... (3 tries left)
==> Downloading https://github.com/oak9io/oak9.cli-public/blob/master/binaries/iac-mac_os-0.11.16.zip
Downloaded to: /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip
SHA256: 87990a2a6f520de56822422fb92d1af145048fe65aba8334b0fc4377822de8f4
==> Retrying download in 8s... (2 tries left)
==> Downloading https://github.com/oak9io/oak9.cli-public/blob/master/binaries/iac-mac_os-0.11.16.zip
Downloaded to: /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip
SHA256: d422877f3dc771ea735189df20d1fb4ddd1f8e12673c30a348b533c59b97e24a
==> Retrying download in 16s... (1 try left)
==> Downloading https://github.com/oak9io/oak9.cli-public/blob/master/binaries/iac-mac_os-0.11.16.zip
Downloaded to: /Users/runner/Library/Caches/Homebrew/downloads/e347a3244523883b886ea3c810675d3c78d8a5b369cfee240d4b715d1178e456--iac-mac_os-0.11.16.zip
SHA256: f9f4155de487267bf34ca6ea8bbbec570f03397b2f840c54d8ea2247f7f72ba2
Warning: Cask reports different sha256: cef527114243a2d48390f8ed454e56cf83f20135511f4f0eabf3dbed660c8ee2
```
See https://github.com/Homebrew/homebrew-cask/issues/89551 for a similar issue.

File: Casks/oak9.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  version "0.11.16"
+  sha256 "cef527114243a2d48390f8ed454e56cf83f20135511f4f0eabf3dbed660c8ee2"
+
+  url "https://github.com/oak9io/oak9.cli-public/raw/master/binaries/iac-mac_os-#{version}.zip",
+      verified: "github.com/oak9io/oak9.cli-public/"
+  name "oak9 CLI"
+  desc "Component of oak9's security compliance SaaS product"
+  homepage "https://oak9.io/"
+
+  auto_updates true
```
Comment (miccal) [inline Casks/oak9.rb:12]: @awright-oak9 are you certain about this?

Comment (miccal) [inline Casks/oak9.rb:12]: @awright-oak9 are you certain about this auto-updating?

Comment (awright-oak9) [reply Casks/oak9.rb]: Sorry I misunderstood what that did. Removed.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @awright-oak9.

**Pull request merged**